x-pack/plugins/observability/server/assets/index_templates/slo_index_templates.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  SLO_COMPONENT_TEMPLATE_MAPPINGS_NAME,
+  SLO_COMPONENT_TEMPLATE_SETTINGS_NAME,
+  SLO_INDEX_TEMPLATE_NAME,
+} from '../common';
+
+export const sloIndexTemplate = {
+  index_patterns: [`${SLO_INDEX_TEMPLATE_NAME}-*`],
+  composed_of: [SLO_COMPONENT_TEMPLATE_MAPPINGS_NAME, SLO_COMPONENT_TEMPLATE_SETTINGS_NAME],
+  priority: 500,
```

Comment:
How did you decide about the priority?

Reply:
I took it from @simianhacker poc.
We might want to make it configurable later on, but for now, we should not have more than one index template, so the priority is not really necessary.